src/main/java/io/stargate/sgv2/jsonapi/service/bridge/executor/NamespaceCache.java

```diff
@@ -0,0 +1,72 @@
+package io.stargate.sgv2.jsonapi.service.bridge.executor;
+
+import com.github.benmanes.caffeine.cache.Cache;
+import com.github.benmanes.caffeine.cache.Caffeine;
+import io.grpc.Status;
+import io.grpc.StatusRuntimeException;
+import io.smallrye.mutiny.Uni;
+import io.stargate.sgv2.jsonapi.config.constants.DocumentConstants;
+import io.stargate.sgv2.jsonapi.exception.ErrorCode;
+import java.time.Duration;
+
+public class NamespaceCache {
```

Comment:
Javadoc? This seems to be container for per-namespace cached information, contained within SchemaCache.
Could explain it here, or in `SchemaCache`, refer one from the other (whichever way makes most sense).